uuid-kinds/src/lib.rs

```diff
@@ -14,15 +14,63 @@ pub use newtype_uuid::{
     GenericUuid, ParseError, TagError, TypedUuid, TypedUuidKind, TypedUuidTag,
 };
 
-#[cfg(feature = "schemars08")]
-use schemars::JsonSchema;
-
 macro_rules! impl_typed_uuid_kind {
     ($($kind:ident => $tag:literal),* $(,)?) => {
         $(
             paste::paste! {
-                #[cfg_attr(feature = "schemars08", derive(JsonSchema))]
-                pub enum [< $kind Kind>] {}
+                pub enum [< $kind Kind >] {}
+
+                #[cfg(feature = "schemars08")]
+                impl schemars::JsonSchema for [< $kind Kind >]{
```

Comment:
Yeah this is about what I expected -- let's chat today about seeing how we can get this into newtype-uuid instead.